sechub-wrapper-owasp-zap/src/main/java/com/mercedesbenz/sechub/zapwrapper/config/ZapScanContext.java

```diff
@@ -326,6 +335,11 @@ public ZapScanContextBuilder setClientCertificateFile(File clientCertificateFile
             return this;
         }
 
+        public ZapScanContextBuilder setHeaderValueFiles(Map<String, File> headerValueFiles) {
+            this.headerValueFiles.putAll(headerValueFiles);
```

Comment:
If somebody calls this method twice what is the situation?

set --> meaning for me (IMHO): "I set this, wipe all existing parts"
--> but it doesn't

So my suggestion:
Either rename to `addHeaderValueFiles` or keep the name but clear always the map before the `putAll` call.